NOTICE

```diff
@@ -8,5 +8,14 @@ within the component is also subject to the terms and conditions
 of these licenses. The LICENSE file contains the text of all the
 licenses which apply within this component.
```

Comment:
This line is missing a section: "The LICENSE file contains the text of all the licenses which apply
within this component, except as indicated in the following paragraphs."

Reply:
Hi mhughesacn ,
Updated the NOTICE & removed the soft links for the images. But for the configuration of new images could you please suggest where do i need to include. 
Thanks